_includes/components/hero.njk

```diff
@@ -1,10 +1,10 @@
 <div class="hero{% if herosmall %} hero-sm{% endif %}">
 	<div class="hero-title">
 		<a href="{{ "/" | url }}" class="hero-link">
-			<h1 class="hero-title-text"><span class="mono">GET</span> <span class="audiowide">CONF</span> <span class="hero-title-date">4/26/19</span></h1>
+			<h1 class="hero-title-text"><span class="mono">GET</span> <span class="audiowide">CONF</span> <span class="hero-title-date">Gender Equity in Tech</span></h1>
 		</a>
-{% if config.register_enabled and showregisterbutton -%}
-		<h2 class="hero-register"><a target="_blank" href="/assets/booklet-forWebsite.pdf" class="btn-primary">Booklet <span>for GET CONF</span></a></h2>
+{% if config.register_enabled and showbooklet -%}
+		<h2 class="hero-register"><a target="_blank" href="/assets/booklet-forWebsite.pdf" class="btn-primary">Booklet <span>for 2019 GET CONF</span></a></h2>
```

Comment:
I added a new variable called showbooklet that we can use.

showregisterbutton also exists and controls registration in other places.